### DISCUSSION TITLE
Add the ability to retrofit old releases.

### DISCUSSION THREAD

**Discussion by James-Jackson-South** [community/community#6098] [Other Feature Feedback, Questions, & Ideas] [answered]
Clearing out the content of an old release and previewing/saving should auto populate the release with PR data.

**Comment by James-Jackson-South** [comment]
Just spotted the almost invisible button. Can we make it look like one please?

![image](https://user-images.githubusercontent.com/40586059/136481707-8f5f7544-c168-42d6-971a-db2da1ecf858.png)

**Comment by MylesBorins** [comment, marked as answer]
Can you expand a bit more on the behavior that you are looking for? Do you have existing releases you want to update? Do you have tags that are not yet releases?

I don't think we would put something in the UI for this specific flow but we can maybe come up with a script to automate this.

**Reply by James-Jackson-South** [reply]
I actually had my understanding wrong as I didn't spot the button. It's possible with the current workflow to simply update existing releases which works well. 

I would however, redesign the UI to make the button more obvious. It fades into the background, especially in dark mode.
